frontend/scripts/react-components/static-content/static-content.component.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import remark from 'remark';
+import remarkReact from 'remark-react';
+import Hero from 'react-components/shared/hero.component';
+import NavSidebar from './nav-sidebar.component';
+
+class StaticContent extends React.PureComponent {
+  render() {
+    const { links = [], content = '', children, notFound } = this.props;
+    const MarkdownContainer = p => (<div className="markdown-content">{p.children}</div>);
+    return (
+      <div className="c-static-content">
+        <Hero className="-read-only" />
+        {!notFound &&
```

Comment:
Kudos for not using trenary operator here!!!!

Reply:
This should probably be two separate components.

Reply:
I knew you'd like it. Food for thought. https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340

Reply:
Hum I don't really agree with that article. The author basically advocates that trenary operators are great because they are less verbose and clutered than if/else/return statements, which they are, but that does not make it easier for humans to understand and debug. Humans many times need help from those "noisy" operators, otherwise we'd all be programming in https://en.wikipedia.org/wiki/Whitespace_(programming_language)

Reply:
hahaha 👍